app/src/main/java/com/example/awesomelanguagelearning/core/data/entity/UserEntity.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.awesomelanguagelearning.core.data.entity
+
+import com.example.awesomelanguagelearning.core.domain.models.User
+import io.realm.kotlin.types.RealmObject
+import io.realm.kotlin.types.annotations.PrimaryKey
+
+class UserEntity(
+    var firstName: String,
+    var lastName: String,
+    var password: String,
+    @PrimaryKey
+    var email: String,
+    var isActive: Boolean
+) : RealmObject {
+
+    constructor() : this("", "", "", "", false)
+
+    override fun toString(): String {
+        return "[$firstName, $lastName, $email, $password, $isActive]"
+    }
+}
+
+fun UserEntity.toDomain(): User =
+    User(
+        firstName = firstName,
+        lastName = lastName,
+        password = password,
+        email = email,
+        isActive = isActive
+    )
```

Comment:
@igel911 you own this class, so there's no reason to add this extension method and better to put it inside `UserEntity` class

Reply:
@slothdeboss The idea was to use this function as an orDefault() one. But then I changed my mind, so there's no need for an extension function.